### ISSUE TITLE
import/named not throwing an error when importing invalid method names from dependencies

### ISSUE THREAD

**Issue by steveharrison** [import-js/eslint-plugin-import#2192]
I expect the following to throw an error that `doesNotExist` is not exported from `react` and so on for each of these lines:
```
import { doesNotExist } from 'react';
import { doesNotExist2 } from 'redux';
import { doesNotExist3 } from '../dist/external';
```

However, no import-related linting error is thrown.

I've provided an example branch here: https://github.com/steveharrison/import-linting-example. To reproduce: run `npm ci`, build an example third-party module with `npm run build:external`, and then run `npm run lint`.

Do you know how I can get invalid import linting errors thrown for all of these lines?

**Comment by ljharb** [comment]
I'm not sure what you mean by "build a third-party module" - you should only be transforming first-party code.

these lines: https://github.com/steveharrison/import-linting-example/blob/master/.eslintrc.js#L24-L28 belong in "rules", not "settings" (and some of them are unnecessary because of the airbnb config)

I'll take a look at the repo tho - it's probably that a rule is failing to pass `commonjs: true`, since react is a CJS package.

**Comment by steveharrison** [comment]
> I'm not sure what you mean by "build a third-party module" - you should only be transforming first-party code.

Yeah, the third-party module example was just to illustrate a very simple transpiled bundle that you'd normally pull in as an NPM module, but the React and Redux examples should suffice since they're also CJS bundles.

> these lines: https://github.com/steveharrison/import-linting-example/blob/master/.eslintrc.js#L24-L28 belong in "rules", not "settings" (and some of them are unnecessary because of the airbnb config)

Ah, whoops, thanks for the pick up! I'll change that in the example; however, it still doesn't work with those changes.

> I'll take a look at the repo tho - it's probably that a rule is failing to pass commonjs: true, since react is a CJS package.

Ah OK, makes sense. That would be great, thanks!

**Comment by steveharrison** [comment]
@ljharb If I want to have a go at debugging whether `commonjs: true` is getting passed through correctly, where should I start to debug this package locally?

**Comment by soryy708** [comment]
Because this mentions transpiled bundles, I wonder if it's related to this one?

- https://github.com/import-js/eslint-plugin-import/issues/3107

Does this still reproduce on latest?
